src/ProductConstructionService/ProductConstructionService.Api/Configuration/ApiRedirection.cs

```diff
@@ -60,10 +60,13 @@ static bool ShouldRedirect(HttpContext ctx)
         {
             return ctx.IsGet()
                 && ctx.Request.Path.StartsWithSegments("/api")
+                && !ctx.Request.Path.StartsWithSegments("/api/status", StringComparison.InvariantCultureIgnoreCase)
+                && !ctx.Request.Path.StartsWithSegments("/api/azdo", StringComparison.InvariantCultureIgnoreCase)
+                && !ctx.Request.Path.StartsWithSegments("/api/codeflow", StringComparison.InvariantCultureIgnoreCase)
```

Comment:
we don't need this too

Reply:
Actually, we do need the azdo and status one

Reply:
yeah but not the codeflow one